app/views/components/bar/test-several-on-page.html

```diff
@@ -0,0 +1,63 @@
+<div class="row">
+  <div class="two-thirds column">
+    <div class="widget">
+      <div class="widget-header">
+        <h2 class="widget-title">Vertical Bar Chart Title 1</h2>
+      </div>
+      <div class="widget-content">
+        <div id="bar-example1" class="chart-container"></div>
+      </div>
+    </div>
+  </div>
+  <div class="two-thirds column">
+    <div class="widget">
+      <div class="widget-header">
+        <h2 class="widget-title">Vertical Bar Chart Title 2</h2>
+      </div>
+      <div class="widget-content">
+        <div id="bar-example2" class="chart-container"></div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<script id="test-script">
+  $('body').on('initialized', function () {
+
+    var dataset1 = [{
+      data: [{
+        name: 'Category A',
+        value: 373,
+        url: 'test',
+        tooltip: 'Tooltip by Data <br> Component A <br> Information'
+      }, {
+        name: 'Category B',
+        value: 372
+      }, {
+        name: 'Category C',
+        value: 236.35
+      }],
+      name: ''
+    }];
+
+    var dataset2 = [{
+      data: [{
+        name: 'Category D',
+        value: 122,
+        url: 'test',
+        tooltip: 'Tooltip by Data <br> Component A <br> Information'
```

Comment:
The tooltip in this example doesn't match the label, just in this case:

![Screen Shot 2019-09-11 at 5 07 17 PM](https://user-images.githubusercontent.com/3249601/64735492-a3473500-d4b6-11e9-9ab7-ab20065a5886.png)

```suggestion
        tooltip: 'Tooltip by Data <br> Component D <br> Information'
```